src/transformers/models/dinov2/__init__.py

```diff
@@ -35,6 +36,18 @@
         "Dinov2Backbone",
     ]
 
+try:
+    if not is_flax_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["modeling_flax_dinov2"] = [
+        "FlaxDinov2ForImageClassification",
+        "FlaxDinov2Model",
+        "FlaxDinov2PreTrainedModel",
+    ]
+
 if TYPE_CHECKING:
     from .configuration_dinov2 import Dinov2Config, Dinov2OnnxConfig
```

Comment:
[![Add docs-internal PRs to the docs-content FR project v2](https://github.com/github/docs/actions/workflows/first-responder-v2-prs-collect.yml/badge.svg?branch=peterbe-patch-1)](https://github.com/github/docs/actions/workflows/first-responder-v2-prs-collect.yml)